src/proto/messaging.proto

```diff
@@ -32,6 +32,16 @@ message PublicKeyBundle {
   PublicKey preKey = 2;
 }
 
+message ContactBundleV1 {
+  PublicKeyBundle keyBundle = 1;
+}
+
+message ContactBundle {
+  oneof version {
+    ContactBundleV1 v1 = 1;
```

Comment:
Couldn't this just be `PublicKeyBundle v1 = 1;` ? It's not clear to me what we gain with adding ContactBundleV1 here.